src/swap/SwapTransactionDetails.tsx

```diff
@@ -0,0 +1,98 @@
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { BottomSheetRefType } from 'src/components/BottomSheet'
+import TokenDisplay from 'src/components/TokenDisplay'
+import Touchable from 'src/components/Touchable'
+import InfoIcon from 'src/icons/InfoIcon'
+import colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { Network } from 'src/transactions/types'
+
+interface Props {
+  network: Network
+  networkFee: BigNumber
```

Comment:
This will probably evolve as it's calculated asynchronously and we'll need a skeleton placeholder I think.

Reply:
yes i will also address this later, since we need the skeleton loader elsewhere on the screen